homeassistant/components/august/subscriber.py

```diff
@@ -49,27 +49,30 @@ def _async_scheduled_refresh(self, now: datetime) -> None:
         """Call the refresh method."""
         self._hass.async_create_task(self._async_refresh(now), eager_start=True)
 
+    @callback
+    def _async_cancel_update_interval(self, _: Event | None = None) -> None:
+        """Cancel the scheduled update."""
+        if self._unsub_interval:
+            self._unsub_interval()
+            self._unsub_interval = None
+
     @callback
     def _async_setup_listeners(self) -> None:
         """Create interval and stop listeners."""
+        self._async_cancel_update_interval()
```

Comment:
I refactored this to be a bit safer in the off chance there was somehow a way we were calling this when the existing one was still there.